weldx/constants.py

```diff
@@ -6,9 +6,15 @@
     preprocessors=[lambda string: string.replace("%", "percent")],  # allow %-sign
     force_ndarray_like=True,
 )
+
 WELDX_QUANTITY = WELDX_UNIT_REGISTRY.Quantity
 
 # add percent unit
 WELDX_UNIT_REGISTRY.define("percent = 0.01*count = %")
 # swap plank constant for hour definition
 WELDX_UNIT_REGISTRY.define("hour = 60*minute = h = hr")
+
+__all__ = (
+    WELDX_UNIT_REGISTRY,
+    WELDX_QUANTITY,
```

Comment:
let's see if the docs build with this
```suggestion
    "WELDX_UNIT_REGISTRY",
    "WELDX_QUANTITY",
```